app/controllers/nwbib/Classification.java

```diff
@@ -421,6 +422,24 @@ private static void collectLabelAndValue(SearchHit hit, JsonNode json,
 		}
 	}
 
+	static String toNwbibNamespace(String id) {
+		return id //
```

Comment:
Hm, do we have left somwhere a "http://purl.org/lobid/nwbib" namespace? I thought these to be obsolete now?

Reply:
Ah, yeah, well, you have explained the why of this in the description of this issue - so never mind.